lib/APIPlugin.js

```diff
@@ -37,17 +37,11 @@ class APIPlugin {
 
 			params.normalModuleFactory.plugin("parser", parser => {
 				Object.keys(REPLACEMENTS).forEach(key => {
-					parser.plugin(`expression ${key}`, expr => {
-						const dep = new ConstDependency(REPLACEMENTS[key], expr.range);
-						dep.loc = expr.loc;
-						parser.state.current.addDependency(dep);
-						return true;
-					});
-
+					parser.plugin(`expression ${key}`, ParserHelpers.toConstantDependency(REPLACEMENTS[key]));
 					parser.plugin(`evaluate typeof ${key}`, ParserHelpers.evaluateToString(REPLACEMENT_TYPES[key]));
 				});
 				IGNORES.forEach(key => {
-					parser.plugin(key, () => true);
+					parser.plugin(key, ParserHelpers.returnTrue);
```

Comment:
If you are extracting this into a helper, it's probably better to describe what it actually does in the context of the parser instead of just "returns true". `ParserHelpers.skipTraversal`?